### DISCUSSION TITLE
Are the backups store in NFS data encrypted?

### DISCUSSION THREAD

**Discussion by frit0-rb** [longhorn/longhorn#7197] [General]
Good morning,

We working with Longhorn years ago and we want to store backups in a NFS, but we want to know if the backups are encrypted or not?

Thanks

**Comment by derekbit** [comment]
No. If you want to encrypt the data block in backupstore, the only solution is using a encrypted volume.
We have a ticket for encypt non-encrypted volume while backing up it. You can follow up https://github.com/longhorn/longhorn/issues/5220.

cc @innobead
